package.json

```diff
@@ -109,7 +109,8 @@
     "mocha": "^8.3.0",
     "mock-stdin": "^1.0.0",
     "playwright": "^1.8.1",
-    "rollup": "^2.39.0"
+    "rollup": "^2.39.0",
+    "typescript": "^4.1.5"
```

Comment:
Might be worth updating to the latest version before landing the PR?

Reply:
Done.